Clarify replicated migration rollback (#21271)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md

1. Validate YBA is up and running with the correct data, including Prometheus.

>     At this point, if you find problems, you can revert to your Replicated installation using the `replicated-migrate rollback` command. Note that any changes made with the new YBA (either using the UI or the API) will not be reflected after the rollback. After the new YBA comes up succesfully, do not attempt to rollback to the original Replicated install of YBA. Rollback is only intended for scenarios where the migration fails.

Reviewer: ```suggestion
    If YBA does not come up or the migration has failed, you can revert to your Replicated installation using the `replicated-migrate rollback` command. Note that any changes made with the new YBA (either using the UI or the API) will not be reflected after the rollback.
    
    After the new YBA comes up succesfully, do not attempt to rollback to the original Replicated install of YBA. Rollback is only intended for scenarios where the migration fails.
```
